recipe/meta.yaml

```diff
@@ -25,6 +27,11 @@ requirements:
   run:
     - python
 
+test:
+  commands:
+    - dir %PREFIX%\pythoncom%CONDA_PY%.dll
```

Comment:
I found an example in https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/meta.yaml#L106